doc/reference_bbox.xml

```diff
@@ -0,0 +1,975 @@
+			<paramdef><type>text </type> <parameter>table_name</parameter></paramdef>
+			<paramdef><type>text </type> <parameter>geocolumn_name</parameter></paramdef>
+			<paramdef><type>boolean </type> <parameter>parent_only</parameter></paramdef>
+		  </funcprototype>
+
+
+		  <funcprototype>
+			<funcdef>box2d <function>ST_EstimatedExtent</function></funcdef>
+			<paramdef><type>text </type> <parameter>schema_name</parameter></paramdef>
+			<paramdef><type>text </type> <parameter>table_name</parameter></paramdef>
+			<paramdef><type>text </type> <parameter>geocolumn_name</parameter></paramdef>
+		  </funcprototype>
+
+		  <funcprototype>
+			<funcdef>box2d <function>ST_EstimatedExtent</function></funcdef>
+			<paramdef><type>text </type> <parameter>table_name</parameter></paramdef>
+			<paramdef><type>text </type> <parameter>geocolumn_name</parameter></paramdef>
+		  </funcprototype>
+		</funcsynopsis>
+	  </refsynopsisdiv>
+
+	  <refsection>
+		<title>Description</title>
+
+		<para>Return the 'estimated' extent of the given spatial table.
+			The estimated is taken from the geometry column's statistics. The
+			current schema will be used if not specified. The default behavior
+			is to also use statistics collected from child tables (tables
+			with INHERITS) if available. If 'parent_only' is set to TRUE, only
+			statistics for the given table are used and child tables are
+			ignored.
+		</para>
+
+		<para>For PostgreSQL&gt;=8.0.0 statistics are gathered by VACUUM
+		ANALYZE and resulting extent will be about 95% of the real
+		one.</para>
+
+                <note>
+                <para>
+In absence of statistics (empty table or no ANALYZE called) this function
+returns NULL.  Prior to version 1.5.4 an exception was thrown
+instead.
+		</para>
+                </note>
+
+
+		<para>For PostgreSQL&lt;8.0.0 statistics are gathered by
+		update_geometry_stats() and resulting extent will be exact.</para>
+
+    <para>Availability: 1.0.0</para>
+    <para>Changed: 2.1.0. Up to 2.0.x this was called ST_Estimated_Extent.</para>
+
+		<para>&curve_support;</para>
+	  </refsection>
+
+
+	  <refsection>
+		<title>Examples</title>
+
+		<programlisting>SELECT ST_EstimatedExtent('ny', 'edges', 'the_geom');
+--result--
+BOX(-8877653 4912316,-8010225.5 5589284)
+
+SELECT ST_EstimatedExtent('feature_poly', 'the_geom');
+--result--
+BOX(-124.659652709961 24.6830825805664,-67.7798080444336 49.0012092590332)
+		</programlisting>
+	  </refsection>
+
+	  <!-- Optionally add a "See Also" section -->
+	  <refsection>
+		<title>See Also</title>
+
+		<para><xref linkend="ST_Extent" /></para>
+	  </refsection>
+	</refentry>
+	<refentry id="ST_Expand">
+	  <refnamediv>
+		<refname>ST_Expand</refname>
+		<refpurpose>Returns a bounding box expanded from another bounding box or a geometry.</refpurpose>
+	  </refnamediv>
+
+	  <refsynopsisdiv>
+		<funcsynopsis>
+		  <funcprototype>
+			<funcdef>geometry <function>ST_Expand</function></funcdef>
```

Comment:
but but but ST_Expand returns rectangular geometry and not a box, if fed with geometry. This minor distinction drove me nuts when I tried to normalize "what the function gets you from which input" graph, after which I decided that boxes are kind of rectangular areas. Also, ST_OrientedEnvelope - it's not a box?

Reply:
Yes, wasn't sure what was best to do with this.  Ideally it would appear under both Geometry and Bounding Box.  Too bad the name is overloaded, but there it is.  

In the end I decided this was more of a bounding box kind of thing.  But can change it back if that's better.

Reply:
And as for ST_OrientedEnvelope, I also find that name confusing.   Perhaps it should be called `ST_OrientedRectangle`?  But it's definitely not a box.   

My take on naming practice is that the support of `box*` types is indicated by using the word "Box" in names.  Other names like `Rectangle` and `Envelope` indicate that a geometry type is used.  The use of the term Envelope at all is perhaps unfortunate, since in PG that function is provided by the `box` types.  But that's what the Simple Features spec uses, so we should use it in a limited way.